### PR TITLE
Update readme to add Pallets Ecosystem notice

### PR DESCRIPTION
**Pull request by samuelhwilliams**
This commit converts the README from `.rst` to `.md`, and then adds the Pallets Community Ecosystem notice. I don't believe `.rst` files can show the same "Important" alert boxes and it feels correct to keep it visually consistent between projects.

I've also updated all of the references in the README from `github.com/flask-admin/flask-admin` to `github.com/pallets-eco/flask-admin`. I can add another commit that does the rest of the project's references, or I can remove that change from this PR to do it all references together in a separate PR.

### CODE REVIEW

**Comment by princerb** [comment]
Is **flask-admin** not active anymore?

**Comment by Abdur-rahmaanJ** [comment]
@princerb Just transferred to the Pallets!

**Comment by princerb** [comment]
> @princerb Just transferred to the Pallets!

I see, but I also can see the date when the last commit was made. So this repo is inactive for this too long, and 78+ open pull requests. For this reason, I am afraid I am not willing to open my pull request while I have some contribution improvements for the project.

**Comment by aenglander** [comment]
[`setup.py`](https://github.com/pallets-eco/flask-admin/blob/master/setup.py#L16) still references `README.rst` for the package's long description.

**Comment by samuelhwilliams** [comment]
@aenglander good spot - have fixed that reference (and the reference in MANIFEST.in as well)

File: setup.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def read(fname):
 
 
 def desc():
-    info = read('README.rst')
+    info = read("README.md")
```
Comment (aenglander) [inline setup.py:17]: [Setting long_description_content_type is required for Markdown](https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/#including-your-readme-in-your-package-s-metadata) as the [spec is to default to RST and fallback to plain text](https://packaging.python.org/en/latest/specifications/core-metadata/#description-content-type)

**Review by aenglander — CHANGES_REQUESTED** [review]

**Review by aenglander — APPROVED** [review]

**Comment by Abdur-rahmaanJ** [comment]
@princerb We actually need more maintainers. Feel free to join the Pallets discord # pallets-eco channel ^^.

**Pull request merged**